src/msw/slider.cpp

```diff
@@ -313,15 +313,28 @@ bool wxSlider::MSWOnScroll(int WXUNUSED(orientation),
     SetValue(newPos);
 
     wxScrollEvent event(scrollEvent, m_windowId);
+    bool          result = false;
```

Comment:
I'd rename it to
```suggestion
    bool          processed = false;
```
because this seems to better describe what it corresponds to.